scripts/demo_example.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import ga4gh.client as client
+import ga4gh.protocol as protocol
+
+httpClient = client.HttpClient("http://server:80/v0.6.e6d6074")
```

Comment:
If this is working off the `develop` branch, you can use the `current` alias instead of having the protocol version in the URL